### PR TITLE
HOTFIX Change drop time source to teuns API

### PR DESCRIPTION
**Pull request by wwhtrbbtt**
I wrote this code in 10 minutes. I have no idea if it works - but it should. 
I added a small function that gets the unix timestamp of a drop from teuns API, instead of scraping NameMC. I replaced the call to `namemc_timing` in `sniper.py`  with a call to the new function, `teun_timing`.

I am sorry if I am wasting time by submitting bad code - this is meant as a base.

### CODE REVIEW

File: mcsniperpy/sniper.py [hunk 1]
```diff
@@ -155,10 +155,15 @@ async def run(self, target=None, offset=None):
                 "init_path"), "accounts.txt")
         )
 
-        droptime = await {"kqzz_api": api_timing, "namemc": namemc_timing}.get(
+        # droptime = await {"kqzz_api": api_timing, "namemc": namemc_timing}.get(
+        #     self.timing_system, "kqzz_api"
+        # )(target, self.session)
+        
+        droptime = await {"kqzz_api": api_timing, "namemc": teun_timing}.get(
```
Comment (gobboo) [inline mcsniperpy/sniper.py:162]: Suggest making a new key for teun api and keep namemc in case it does come back up, or teun goes down and namemc is up.

Comment (gobboo) [reply mcsniperpy/sniper.py:162]: Also make sure you test it and it works, check response and see if it is hitting teuns api

Comment (Kqzz) [reply mcsniperpy/sniper.py:162]: Agreed on both of those things, Gobbo.

Comment (Kqzz) [reply mcsniperpy/sniper.py:162]: NameMC timing should definitely still exist

**Review by Kqzz — CHANGES_REQUESTED** [review]
This function should definitely make use of the custom `RequestManager` class used for most http requests in MCsniperPY. also, it should make use of the custom input function too. Both _should_ already be imported into `name_system.py`.

File: mcsniperpy/sniper.py (hunk 1 above)
Comment (wwhtrbbtt) [reply mcsniperpy/sniper.py:162]: I did test the `teun_timing` function - it works. I don't quite know how the config system works, but I guess you can change where it gets the timing from? I agree - nameMC timing should still exist.

File: mcsniperpy/util/name_system.py [hunk 2]
```diff
@@ -69,17 +69,28 @@ async def namemc_timing(username: str, session: RequestManager) -> float:
...
+        return resp_json["UNIX"]
+    else:
+        log.error(
+            f"failed to parse droptime for {color.l_cyan}{username}{color.reset} through teuns's MC API"
+        )
+        log.error(f"{resp_json['error']} | {resp.status}")
+        close(0)
+
+    close(0)
+    return 0
```
Comment (Kqzz) [inline mcsniperpy/util/name_system.py]: I'm not sure how teuns api works, but does this work if teun's API needs the name of the old owner?

Comment (wwhtrbbtt) [reply mcsniperpy/util/name_system.py]: He has a bypass AFAIK and is caching new names right now

Comment (Kqzz) [reply mcsniperpy/util/name_system.py]: oh awesome

**Pull request merged**